packages/generator/generators/react-component/index.js

```diff
@@ -106,5 +130,49 @@ module.exports = yeoman.Base.extend({
 				this
 			);
 		}
+		if (this.props.parentIndex) {
+			const indexPath = path.join(this.props.path, './index.js');
+			let code;
+			try {
+				code = fs.readFileSync(indexPath);
+			} catch (error) {
+				this.log(error.message);
+			}
+			if (code) {
+				const ast = recast.parse(code);
```

Comment:
Could you change the variable's name to be more explicit?

Reply:
mostly `b` below

Reply:
parsedIndex ?